workflows/differentialabundance.nf

```diff
@@ -259,16 +258,16 @@ workflow DIFFERENTIALABUNDANCE {
     ch_citations_file = Channel.from(citations_file)
 
     ch_report_input_files = ch_all_matrices
-        .tail()
+        .map{tuple(it[1], it[2], it[3])}
```

Comment:
```suggestion
        .map{ it.tail() }
```
I think I see what you were doing there (my .`.tail()` wasn't right, though it didn't break anything).

But this is a simpler way of removing the meta.